### ISSUE TITLE
Problem with Custom Filter

### ISSUE THREAD

**Issue by b-morgan** [Noshei/GatheringTracker#26]
On my Tailor / Engineer I added two items in the Custom Filter, 222804 (Weavercloth Bolt) and 219150 (Pile of Rusted Scrap). Both items drop from mobs and both show up in the list on screen but when I open options I get the following error. 

If you need more information, let me know.

```
1x GatheringTracker/Config.lua:1544: attempt to compare string with number
[string "@GatheringTracker/Config.lua"]:1544: in function `member'
[string "@AnnoyingPopupRemover/Libs/AceConfigDialog-3.0-87/AceConfigDialog-3.0.lua"]:215: in function <...ver/Libs/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:164>
[string "@AnnoyingPopupRemover/Libs/AceConfigDialog-3.0-87/AceConfigDialog-3.0.lua"]:1206: in function <...ver/Libs/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:1113>
[string "@AnnoyingPopupRemover/Libs/AceConfigDialog-3.0-87/AceConfigDialog-3.0.lua"]:1642: in function `FeedGroup'
[string "@AnnoyingPopupRemover/Libs/AceConfigDialog-3.0-87/AceConfigDialog-3.0.lua"]:1567: in function <...ver/Libs/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:1551>
[string "=[C]"]: ?
[string "@Ace3/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <Ace3/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Ace3/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Ace3/AceGUI-3.0-41/widgets/AceGUIContainer-TabGroup.lua"]:342: in function `SelectTab'
[string "@Ace3/AceGUI-3.0-41/widgets/AceGUIContainer-TabGroup.lua"]:200: in function <...Ace3/AceGUI-3.0/widgets/AceGUIContainer-TabGroup.lua:197>

Locals:
data = <table> {
}
imageSize = <table> {
 1 = 24
 2 = 24
}
border = <table> {
}
borderColor = <table> {
}
overlay = <table> {
}
(*temporary) = "219150"
(*temporary) = 2
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to compare string with number"
id = "219150"
GT = <table> {
 expansionsOrder = <table> {
 }
 modules = <table> {
 }
 count = <table> {
 }
 media = <table> {
 }
 db = <table> {
 }
 NotificationPause = true
 Options = <table> {
 }
 categoriesOrder = <table> {
 }
 Notifications = <table> {
 }
 DebugCount = 0
 name = "GatheringTracker"
 IDs = <table> {
 }
 PlayerEnteringWorld = false
 Pools = <table> {
 }
 baseName = "GatheringTracker"
 waitTable = <table> {
 }
 defaultModuleState = true
 priceSources = <table> {
 }
 TSM = ""
 groupMode = "WHISPER"
 Enabled = true
 Player = "Lartt"
 metaData = <table> {
 }
 defaults = <table> {
 }
 ItemData = <table> {
 }
 categories = <table> {
 }
 GlobalStartTime = 1729512193
 defaultModuleLibraries = <table> {
 }
 sender = <table> {
 }
 orderedModules = <table> {
 }
 baseFrame = <table> {
 }
 enabledState = true
 Display = <table> {
 }
 expansions = <table> {
 }
 InventoryData = <table> {
 }
}
```

**Comment by Noshei** [comment]
hmm, I'll have to look at this, and probably add those to the normal filters as well.

**Comment by Noshei** [comment]
ah, I dont typically add the tailoring bolts, since they are typically crafted.  The rusted scrap I can probably add though (rare to have an engineering specific item drop)

**Comment by Noshei** [comment]
ah, I see why I didn't add Pile of Rusted Scrap, it can be bought from vendors.

**Comment by b-morgan** [comment]
I'm not sure you should add either the bolts or the scrap to the regular lists. I'm just keeping track because they drop for that toon.

**Comment by Noshei** [comment]
resolved in v2.6.3
